gammapy/makers/utils.py

```diff
@@ -185,11 +217,58 @@ def _map_spectrum_weight(map, spectrum=None):
     return map * weights.reshape(shape.astype(int))
 
 
+def _evaluate_bkg(pointing, ontime, bkg, geom, use_region_center, obstime, d_omega):
+    """
+    Evaluate the background IRF on a given geometry.
+
+    Parameters
+    ----------
+    pointing :  `~gammapy.data.FixedPointingInfo` or `~astropy.coordinates.SkyCoord`
+        Observation pointing.
+    ontime : `~astropy.units.Quantity`
+        Observation ontime. i.e. not corrected for deadtime
```

Comment:
You mean the observation duration?

Reply:
I re-used the docstring of `make_map_background_irf`. 
It is indeed the observation duration, which is also names ontime in the tables from datastores if I recall correctly.